src/frontend/js/components/DashBoard/index.tsx

```diff
@@ -1,5 +1,17 @@
+import { BrowserRouter } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import DashBoardRouter from 'components/DashBoard/Router';
+import getBasename from './utils';
+import useDashboardRoutes from '../../hooks/useDashboardRoutes';
```

Comment:
You could absolute path to import `hooks/useDashboardRoutes` module.

```suggestion
import DashBoardRouter from 'components/DashBoard/Router';
import useDashboardRoutes from 'hooks/useDashboardRoutes';
import getBasename from './utils';
```